www/src/data/blog-posts/astro-021-release.md

```diff
@@ -0,0 +1,43 @@
+---
+title: 'Astro 0.21'
+author: 'fred'
+description: 'Astro v0.21.0 is finally here!'
+publishDate: 'November 19, 2021'
+permalink: 'https://astro.build/blog/astro-021-release'
+lang: 'en'
+heroImage: '/assets/blog/astro-021-preview/hero.png'
+heroImageAlt: 'Spacecraft overlooking Earth'
+---
+
+Astro v0.21.0 is finally here! This is by far our biggest release ever, thanks to a ground-up rewrite of major Astro internals. After months of development and public testing, we are so excited to get this new version of Astro into your hands.
+
+- A new build engine, **powered by [Vite](http://vitejs.dev/)**
+- A new Astro compiler, **written in [Go](https://golang.org/)**
+- Components in Markdown (like MDX, but better!)
+- Improved errors, stack traces, HMR, and overall dev experience
+- A grab-bag of other new features for you to explore
+
+Check out our original [v0.21 Early Preview](/blog/astro-021-preview) for even more details on what you can expect from this release.
+
+[Try Astro v0.21.0 today](https://astro.new) right in your browser. Or, run `npm install astro@latest` in any new or existing project directory to get started. 
+
+
+## Migrate to v0.21
+
+Very little has changed in the v0.21.0 API to make your migration as easy as possible. Check out the new [v0.21 migration guide](https://docs.astro.build/migration/0.21.0) for a full breakdown of meaningful changes.
```

Comment:
```suggestion
Very little has changed in the v0.21.0 API to make your migration as easy as possible. Check out the new [v0.21 migration guide](/migration/0.21.0) for a full breakdown of meaningful changes.
```

Reply:
correct as-is: the blog and docs sites are on different domains